From 8f26b50e0710ecfe42df7d8f3089ce65480a741e Mon Sep 17 00:00:00 2001
From: Mathieu Westphal <mathieu.westphal@kitware.com>
Date: Thu, 20 Jun 2024 13:48:37 +0200
Subject: [PATCH] hdf5: Add missing method from mangling

hdf5 mangling was missing a few methods which caused segfault when
using a local python and h5py, this fixes it by adding them.
---
 ThirdParty/hdf5/vtkhdf5/src/vtk_hdf5_mangle.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ThirdParty/hdf5/vtkhdf5/src/vtk_hdf5_mangle.h b/ThirdParty/hdf5/vtkhdf5/src/vtk_hdf5_mangle.h
index 2be50a711d..7ff64f02ae 100644
--- a/VTK/ThirdParty/hdf5/vtkhdf5/src/vtk_hdf5_mangle.h
+++ b/VTK/ThirdParty/hdf5/vtkhdf5/src/vtk_hdf5_mangle.h
@@ -3138,6 +3138,8 @@
 #define H5_time_t_reg_free_list vtkhdf5_H5_time_t_reg_free_list
 #define H5T_init vtkhdf5_H5T_init
 #define H5T__init_native vtkhdf5_H5T__init_native
+#define H5T__init_native_internal vtkhdf5_H5T__init_native_internal
+#define H5T__init_native_float_types vtkhdf5_H5T__init_native_float_types
 #define H5T__insert vtkhdf5_H5T__insert
 #define H5Tinsert vtkhdf5_H5Tinsert
 #define H5T_invoke_vol_optional vtkhdf5_H5T_invoke_vol_optional
-- 
2.39.3

